.vscode/c_cpp_properties.json

```diff
@@ -1,19 +1,19 @@
 {
     "env": {
-        "BOLOS_SDK": "/opt/nanos-secure-sdk",
+        "BOLOS_SDK": "/opt/nanosplus-secure-sdk",
         "BOLOS_ENV": "/opt/bolos-devenv"
     },
     "configurations": [
```

Comment:
Why is this list hardcoded?
This should come from the extension?

Reply:
Not sure, I suppose that this list contains the default value at Vscode start-up
In any case, cannot be nanos